backend/wmg/api/query.py

```diff
@@ -22,9 +22,28 @@ class WmgQueryCriteria(BaseModel):
     sex_ontology_term_ids: List[str] = Field(default=[], unique_items=True, min_items=0)
 
 
+class FmgQueryCriteria(BaseModel):
+    organism_ontology_term_id: str  # required!
+    tissue_ontology_term_ids: List[str] = Field(default=[], unique_items=True, min_items=0)
```

Comment:
ooo sorry I missed this. Don't use default=[]. You need to do `Field(default_factory=list)`. This is because we want a new list to be created as the default. If you don't do this, then the same list will be used as default for all instances of `FmgQueryCriteria`. Apply this same logic every where you are using a `dict` or `list` as the default value.

Reply:
Should this change be applied to `WmgQueryCriteria` as well?

Reply:
Also, does this matter if the criteria will never be modified in place?

Reply:
> Should this change be applied to WmgQueryCriteria as well?
yes for both

> Also, does this matter if the criteria will never be modified in place?
That makes it less of a problem, but could lead to some difficult debugging in the future if we don't fix it now.